README.md

```diff
@@ -1 +1,38 @@
-# Hello-world
\ No newline at end of file
+Hello Microverse
+
+This Project is for greeting to Microverse and is done on second day of Microverse Bootcamp
+
+Description the project.
+
+Built With
+HTML&CSS
+
+Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+Authors
+bust_in_silhouette Author1
+
+GitHub: @pazzo97
+Twitter: @twitterhandle
+LinkedIn: Patrick MUKUNZI
```

Comment:
Kindly make this information as a hyperlink, you can do that by surrounding username with [] then put the link between ().
```
[username](url)
```